### PR TITLE
Make GSOC page

### PR DESCRIPTION
**Pull request by andronovhopf**

### CODE REVIEW

File: content/blog/2020-02-04-gsoc-ideas.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: Join DVC for Google Summer of Code
```
Comment (shcheklein) [inline content/blog/2020-02-04-gsoc-ideas.md]: should we put 2020?

File: content/blog/2020-02-04-gsoc-ideas.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+   cloud storages (like GCS or S3). We need to identify and implement model zoos
+   (think PyTorch Hub, the Caffe Model Zoo, or the TensorFlow DeepLab Model Zoo)
+   and data registries for types that are not supported by DVC yet. Currently,
+   we’ve tested `summon` with PyTorch image segmentation models and Pandas
+   dataframes. We’re looking for students to explore other possible use cases.
+   <br /> <br /> _Skills required:_ Python, Git, and some machine learning or
+   data science experience <br /> _Difficulty rating:_ Beginner-Medium <br />
+
+4. **Continuous delivery for JetBrains TeamCity.** Continuous integration and
+   continuous delivery (CI/CD) for ML projects is an area where we see DVC make
```
Comment (shcheklein) [inline content/blog/2020-02-04-gsoc-ideas.md]: can we put a link to the martin fowler blog post? to give more context `[DVC make big impact](...)`

Comment (andronovhopf) [reply content/blog/2020-02-04-gsoc-ideas.md]: definitely

File: content/blog/2020-02-04-gsoc-ideas.md (hunk 1 above)
Comment (andronovhopf) [reply content/blog/2020-02-04-gsoc-ideas.md]: yes

**Pull request merged**